bin/test-clouds-cleanup

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+#
+# Cleans up integration tests from 4 cloud providers:
+# - Amazon (EKS)
+# - DigitalOcean (DO)
+# - Google (GKE)
+# - Microsoft (AKS)
+#
+# This script assumes you have a working Kubernetes cluster set up on each Cloud
+# provider, and that Kubernetes contexts are configured via environment
+# variables.
+
+set -eu
+
+for CLUSTER in $AKS $DO $EKS $GKE
```

Comment:
these variables appear to be assumed, but we haven't documented that we need to set them anywhere; consider a more helpful error message

```
$ bin/test-clouds `pwd`/bin/linkerd
Checking the linkerd binary...[ok]
bin/test-clouds: line 39: AKS: unbound variable
```